src/models/queries/signup.js

```diff
@@ -0,0 +1,43 @@
+const connection = require('../database/db_connection');
+const bcrypt = require("bcrypt");
+
+const signup = (data, callback) => {
+  var salt = bcrypt.genSaltSync(10);
+  const passwordHash = bcrypt.hashSync(data.password, salt);
+  const sql = {
+    text: "INSERT INTO users (name,password,email,role) VALUES ($1,$2,$3,$4)",
+    values: [`${data.username}`, `${passwordHash}`, `${data.email}`, 'user']
+  };
+  connection.query(sql, errRegister => {
+    if (errRegister) {
+      callback(errRegister);
+    } else {
+      callback(null, true);
+    }
+  });
+};
+
+const signupOnwer = (data, callback) => {
+  var salt = bcrypt.genSaltSync(10);
+  const passwordHash = bcrypt.hashSync(data.password, salt);
+  const sql = {
```

Comment:
it's better to have only database queries inside the query files, and to make the hash in separate file and then in the
 ``` signup.js ```  file require the hash file and signup query file :bulb:

Reply:
humm , ok when we set together will change it @Salam-Dalloul :1234:

Reply:
its very good :smile: @Salam-Dalloul  , I told him :+1:  . 
i thinks its better to create new folder "helper" , its contain all helper function like Hash password and compare , and so on :+1: 

@MahmoudMH  done it in the next time :100: